7-CSS-Box-Model/Part-2/css/style.css

```diff
@@ -0,0 +1,118 @@
+* {
+  margin: 0;
+  padding: 0;
+  transition: all 0.1s ease-in;
+}
+
+section {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  font-size: 0%;
+  box-sizing: border-box;
+  border: solid 15px #2b81ac;
+  background-color: #3399cc;
+  color: white;
+  font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+}
+
+span {
+  font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+}
+
+h1 {
+  font-size: 25px;
+  margin: 2% 20%;

```

Comment:
this is incorrect, you could wrap h1 and blocks with section 
<section><h1></h1><div><div></section> section {width: 80%;margin: o auto} and that's it - your text would be on the left without margin it